apiExamples/roundedRightAlign.js

```diff
@@ -0,0 +1,11 @@
+export function roundedRightAlignExample() {
+  const rightAlignElem = document.querySelector('#rightAlignElem');
+
+  rightAlignElem.addEventListener('mouseover', () => {
+    rightAlignElem.iconOnly = false;
+  });
+
+  rightAlignElem.addEventListener('mouseout', () => {
+    rightAlignElem.iconOnly = true;
+  });
```

Comment:
Include the focus events from the left align example as well.

Reply:
Changes were made under this commit: [docs(rounded): add examples for rounded button](https://github.com/AlaskaAirlines/auro-button/pull/214/commits/3ba91aa69416d83385db790b3cde6264023b3f42) https://github.com/AlaskaAirlines/auro-button/issues/212